recipes/rte-rrtmgp/bld.bat

```diff
@@ -0,0 +1,45 @@
+@echo off
+:: Stop on any error
+setlocal ENABLEEXTENSIONS
+
+set BUILD_DIR=build
+set BUILD_TYPE=Debug
+
+set "HOST=x86_64-w64-mingw32"
+set "FC=%HOST%-gfortran.exe"
+set FCFLAGS="-ffree-line-length-none -m64 -std=f2008 -march=native -fbounds-check -fmodule-private -fimplicit-none -finit-real=nan -fbacktrace"
```

Comment:
@makepath-alex as per https://github.com/earth-system-radiation/rte-rrtmgp/pull/360 should we remove these flags?

Reply:
As we discussed, we will remove them for a subsequent release. For now let them stay